docs/install/install-node.rst

```diff
@@ -4,166 +4,99 @@
 
-Finally, you will need to run the ``flocker-ca`` tool that is installed as part of the CLI package.
-This tool generates TLS certificates that are used to identify and authenticate the components of your cluster when they communicate, which you will need to copy over to your nodes.
-Please see the :ref:`cluster authentication <authentication>` instructions.
+#. **Log into the first node as root.**
 
-.. _ubuntu-14.04-install:
+   .. prompt:: bash $
 
-Installing on Ubuntu 14.04
-==========================
+      ssh root@<your-first-node>
 
-.. note:: The following commands all need to be run as root on the machine where ``clusterhq-flocker-node`` will be running.
+#. **Install the** ``clusterhq-flocker-node`` **package.**
 
-Setup the pre-requisite repositories and install the ``clusterhq-flocker-node`` package.
+   To install ``clusterhq-flocker-node`` on Ubuntu 14.04 you must install the package provided by the ClusterHQ repository.
+   The commands below will install the two repositories and the ``clusterhq-flocker-node`` package.
+   
+   Paste the following commands into a root console on the target node:
+   
+   .. task:: install_flocker ubuntu-14.04
+      :prompt: [root@ubuntu]#
 
-.. task:: install_flocker ubuntu-14.04
-   :prompt: [root@ubuntu]#
+#. **Repeat steps 1 and 2 on all other nodes.**
 
-Flocker's container management features depend on Docker.
-Make sure `Docker (at least 1.8) is installed`_  and running.
+   If you haven't already, log into your other nodes as root, and then run step 2 until all the nodes in your cluster have installed the ``clusterhq-flocker-node`` package.
 
-Finally, you will need to run the ``flocker-ca`` tool that is installed as part of the CLI package.
-This tool generates TLS certificates that are used to identify and authenticate the components of your cluster when they communicate, which you will need to copy over to your nodes.
-Please continue onto the next section, with the cluster authentication instructions.
+.. note:: Flocker's container management features depend on Docker.
+          You will need to make sure `Docker (at least 1.8) is installed`_ and running.
 
 Next Step
 =========
 
-In the next step :ref:`the node control and agent services will be configured and started<post-installation-configuration>`.
+You are now ready to :ref:`install the Flocker plugin for Docker<install-docker-plugin>`, which allows Flocker to manage your data volumes while using other tools such as Docker, Docker Swarm, or Mesos to manage your containers.
+
+Alternatively, you can go ahead to the next section, where you will need to  :ref:`configure your cluster<post-installation-configuration>`, starting with setting up authentication so the different parts of Flocker can communicate.
```

Comment:
As a user I have no idea why I would choose the second one.

"where you will need to" - maybe something more like "which explains how to".